storage/backend/backends.go

```diff
@@ -26,10 +33,7 @@ var SnapshotDrivers = map[string]func() (storage.SnapshotDriver, error){
 // NewMountDriver instantiates and return a mount driver instance of the
 // specified type
 func NewMountDriver(backend, mountpath string) (storage.MountDriver, error) {
-	f, ok := MountDrivers[backend]
-	if !ok {
-		return nil, errored.Errorf("invalid mount driver backend: %q", backend)
-	}
+	f, _ := MountDrivers[backend]
```

Comment:
You can omit the second value from the assignment entirely if you're not checking for existence when plucking a value from a map.

```go
f := MountDrivers[backend]
```

If it doesn't exist, you just get the default uninitialized state for the type:  https://play.golang.org/p/HXax59236C

Why are we switching from checking to see if the backend exists to assuming it exists?  I'm not familiar with this code yet, so I'm just curious.